Foundation/NSString.swift

```diff
@@ -228,7 +228,12 @@ public class NSString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, N
     
     override internal var _cfTypeID: CFTypeID {
         return CFStringGetTypeID()
-    }
+  }
+  
+  public override func isEqual(object: AnyObject?) -> Bool {
+    guard let string = object as? String else { return false }
```

Comment:
object as? String however will not work on linux: it will need to be changed from the implicit type conversion to an explicit version ala (object as? NSString)._swiftObject